x-pack/legacy/plugins/siem/common/constants.ts

```diff
@@ -6,6 +6,11 @@
 
 export const APP_ID = 'siem';
 export const APP_NAME = 'SIEM';
+export const DEFAULT_BYTES_FORMAT = 'format:bytes:defaultPattern';
+export const DEFAULT_DATE_FORMAT = 'dateFormat';
+export const DEFAULT_DATE_FORMAT_TZ = 'dateFormat:tz';
+export const DEFAULT_DARK_MODE = 'theme:darkMode';
 export const DEFAULT_INDEX_KEY = 'siem:defaultIndex';
 export const DEFAULT_ANOMALY_SCORE = 'siem:defaultAnomalyScore';
 export const DEFAULT_MAX_TABLE_QUERY_SIZE = 10000;
+export const DEFAULT_SCALE_DATE_FORMAT = 'dateFormat:scaled';
```

Comment:
is this used?

Reply:
I do not think so, I just copy like an idiot what there before checking. Thanks!!!